static/js/modules/send-message.js

```diff
@@ -150,26 +151,26 @@ var _ = require('underscore'),
     $modal.find('.has-error').removeClass('has-error');
     $modal.find('.help-block').text('');
 
-    var val = [],
+    var initialTo,
         to = options.to,
         message = options.message || '';
 
     if (to) {
       if (typeof to === 'string') {
-        val.push(to);
+        initialTo = [to];
```

Comment:
This was a refactoring of original code, so I just attempted a rename for clarity. FWIW I'm massively not a fan of reusing variables, but it does seem to be the JS way, so that's neither here nor there.